ustriage/ustriage.py

```diff
@@ -763,6 +808,14 @@ def launch():
                         help='Mark bugs touched more recently than this many'
                              ' days (default disabled in triage, 7 days in '
                              ' tag/subscription search)')
+    parser.add_argument('-S', '--save-tagged-bugs',
+                        default=False,
+                        dest='filename_save',
+                        help='Save the list of reported tagged bugs to %file')
+    parser.add_argument('-C', '--compare-tagged-bugs-to',
+                        default=False,
+                        dest='filename_compare',
+                        help='Compare the reported tagged bugs to %file')
```

Comment:
The `%` in `%file` break `--help`, the error is `TypeError: must be real number, not dict`, but I think that's a bit bogus and the real reason is that the `%` is interpreted as a format string. I removed the `%` to verify that's the culprit and it works just fine.

Reply:
Thanks, fixed now